<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">

<!--Converted with LaTeX2HTML 2002-2-1 (1.70)
original version by:  Nikos Drakos, CBLU, University of Leeds
* revised and updated by:  Marcus Hennecke, Ross Moore, Herb Swan
* with significant contributions from:
  Jens Lippmann, Marek Rouchal, Martin Wilck and others -->
<HTML>
<HEAD>
<TITLE>program</TITLE>
<META NAME="description" CONTENT="program">
<META NAME="keywords" CONTENT="program">
<META NAME="resource-type" CONTENT="document">
<META NAME="distribution" CONTENT="global">

<META NAME="Generator" CONTENT="LaTeX2HTML v2002-2-1">
<META HTTP-EQUIV="Content-Style-Type" CONTENT="text/css">

<LINK REL="STYLESHEET" HREF="program.css">

<LINK REL="next" HREF="node1.html">
</HEAD>

<BODY >
<!--Navigation Panel-->
<A NAME="tex2html2"
  HREF="node1.html">
<IMG WIDTH="37" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="next"
 SRC="file:/usr/local/share/lib/latex2html/icons/next.png"></A> 
<IMG WIDTH="26" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="up"
 SRC="file:/usr/local/share/lib/latex2html/icons/up_g.png"> 
<IMG WIDTH="63" HEIGHT="24" ALIGN="BOTTOM" BORDER="0" ALT="previous"
 SRC="file:/usr/local/share/lib/latex2html/icons/prev_g.png">   
<BR>
<B> Next:</B> <A NAME="tex2html3"
  HREF="node1.html">SITEBLAST commandline options</A>
<BR>
<BR>
<!--End of Navigation Panel-->
<FONT SIZE="+1"><B>Genomic sequence alignment with motif anchors.</B></FONT>

<H3>Abstract:</H3>
<DIV>
<FONT SIZE="+1">Motivation:</FONT> Comparative sequence analysis is the essence of many approaches to genome annotation. Heuristic alignment algorithms utilize similar seed pairs to anchor an alignment. Some applications of local alignment algorithms (e.g. phylogenetic footprinting) would benefit from including prior knowledge (e.g. binding site motifs) in the alignment building process.

<BR><FONT SIZE="+1">Results:</FONT> We introduce predefined sequence patterns as anchor points into a heuristic local alignment strategy. We extended the BLASTZ program for this purpose. A set of seed patterns is either given as consensus sequences in IUPAC code or position-weight-matrices (PWMs). Phylogenetic footprinting of promoter regions is one of many potential application for the SITEBLAST software.
</DIV>
<P>
<BR><HR>
<!--Table of Child-Links-->
<A NAME="CHILD_LINKS"></A>

<UL>
<LI><A NAME="tex2html4"
  HREF="node1.html">SITEBLAST commandline options</A>
<UL>
<LI><A NAME="tex2html5"
  HREF="node2.html">Switch A, Printing option</A>
<LI><A NAME="tex2html6"
  HREF="node3.html">Switch I, IUPAC seeds - key word tree search</A>
<LI><A NAME="tex2html7"
  HREF="node4.html">Switch i, IUPAC seeds - trivial search</A>
<LI><A NAME="tex2html8"
  HREF="node5.html">Switch D, Distance to IUPAC seeds.</A>
<LI><A NAME="tex2html9"
  HREF="node6.html">Switch P, Seed search with position-weight-matrices</A>
<LI><A NAME="tex2html10"
  HREF="node7.html">Switch pValue(required) and powerLimit(optional)</A>
<LI><A NAME="tex2html11"
  HREF="node8.html">Switch power(required) and pValueLimit(optional)</A>
</UL>
<BR>
<LI><A NAME="tex2html12"
  HREF="node9.html">Copyright and license agreement</A>
<LI><A NAME="tex2html13"
  HREF="node10.html">About this document ...</A>
</UL>
<!--End of Table of Child-Links-->
<BR><HR>
<ADDRESS>
Christoph Dieterich
2004-12-06
</ADDRESS>
</BODY>
</HTML>
